src/connectionconfig/connectionDialogWebviewController.ts

```diff
@@ -119,9 +124,15 @@ export class ConnectionDialogWebviewController extends ReactWebviewPanelControll
         }
 
         this.registerRpcHandlers();
-        this.initializeDialog().catch((err) =>
-            vscode.window.showErrorMessage(getErrorMessage(err)),
-        );
+        this.initializeDialog().catch((err) => {
+            void vscode.window.showErrorMessage(getErrorMessage(err));
+            sendErrorEvent(
+                TelemetryViews.ConnectionDialog,
+                TelemetryActions.Initialize,
+                err,
```

Comment:
No potential PIIs?

Reply:
The spots in `initializeDialog()` that handle potential PII have their own error catches that emit error telemetry, and those spots have `includeErrorMessage` set to `false`.  Everything else during initialization shouldn't have PII.